docs/userguide.md

```diff
@@ -201,9 +207,63 @@ Supported dependency types:
 - `maven` - Maven dependencies that can be found at many different sources. Specified as `package: javax.xml.bind:jaxb-api` where the convention is `groupId:artifactId`.
 - `npm` - Javascript dependencies found at [npmjs.com](https://www.npmjs.com/).
 - `nuget` - .NET dependencies found at [NuGet.org](https://www.nuget.org/).
-- `python` - Python dependencies found at [Pypi.org](https://pypi.org/).
+- `python` - Python dependencies that are typically found at [Pypi.org](https://pypi.org/).
```

Comment:
This was not changed from Zach's original PR.  I prefer calling it `python`, since we fetch it from `pypi` (via `pip`, `pipenv`, or eventually `poetry`, which does not use pip at all), not from `pip` specifically.  The use of `pip` is an implementation detail, and one that most people probably don't care about.  Python dependencies are like nodejs dependencies, they are unaware of package manager at runtime, they just get picked up because they're in the right place.

If we change it, it should be `pypi` instead of python.

Reply:
I think `pypi` is actually more accurate. I'm fine if you change it to that Wes. I really don't like `pip` because that really doesn't clarify anything, it only describes what we do in the backend with the fetcher.